tokens/_options/deprecated-2020-fall.json5

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
@lancetharper @mhewson do we need to deprecate the options since they're internal only? can we just delete or smth instead?

Reply:
Yeah, we can just delete them if nothing is still using them as they aren't exported for consumers

Reply:
oh i guess we do need these options available to continue compiling the existing stuff. LGTM